services/portfolios_ca/src/types/targets.ts

```diff
@@ -1,5 +1,5 @@
 export interface TargetsServiceI {
-  getPortfoliosByTarget: () => Promise<PortfoliosByTarget>;
+  getPortfoliosByTarget: (targetId: number) => Promise<Array<PortfoliosByTarget>>;
```

Comment:
the dependency injection container it's not fully complying with the type, I'll check it out later. But this function actually returns just the object not an array

Reply:
Solved